design/README.md

```diff
@@ -0,0 +1,32 @@
+# HackWPI 2018 Design
+Design documents for HackWPI 2018
+
+## Preview
+
+![Home](Home.jpg)
+
+![Day of](Day-of.jpg)
+
+![Map](Map.jpg)
+
+## Editing
+
+You need a Mac with [Sketch](https://www.sketchapp.com) installed to open the document.
+
+## Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or Facebook group chat with the owners of this repository before making a change.
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+
+3. Increase the version numbers in any examples files and the README.md to the new version that this Pull Request would represent. The versioning scheme we use is SemVer.
+
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
```

Comment:
Lol we don't have enough developers to have two reviewers. You can probably just remove the whole number

Reply:
Talked this over with Harry, this is specifically for design documents, so both Harry and I can review them.